python/ray/_private/accelerators/tpu.py

```diff
@@ -32,9 +32,8 @@
 
 NOSET_TPU_VISIBLE_CHIPS_ENV_VAR = "RAY_EXPERIMENTAL_NOSET_TPU_VISIBLE_CHIPS"
 
-# TPU VMs come with 4 chips per host and 2 tensorcores per chip.
+# By default TPU VMs come with 4 chips per host and 2 tensorcores per chip.
 # For more details: https://cloud.google.com/tpu/docs/system-architecture-tpu-vm
-TPU_NUM_CHIPS_PER_HOST = 4
 TPU_CORES_PER_CHIP = 2
```

Comment:
Sorry one more thing, `TPU_CORES_PER_CHIP = 2` won't be true for v5e. We need to modify the logic [here](https://github.com/ray-project/ray/blob/0c25da892689eef8c81137044103c03ddc75e1a2/python/ray/autoscaler/_private/gcp/config.py#L127) to handle the v5litepod case.

We might be able to just remove this constant altogether and handle that in the config.py file

Reply:
Applied more fixes, please take a look.